src/System.ComponentModel.Annotations/tests/System/ComponentModel/DataAnnotations/MetadataTypeAttributeTests.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.ComponentModel.DataAnnotations.Tests
+{
+    public class MetadataTypeAttributeTests
```

Comment:
@danmosemsft I think that I've addressed the [code review](https://github.com/dotnet/corefx/pull/33312/files/50276f253aa577d0d776eea8f3cc4e02d7028f19#diff-dd8e59029d6ee936017fa4394151a32b)
I'm willing to get your feedback for this.
Thanks.